unblob/extractors/command.py

```diff
@@ -28,7 +28,8 @@ def extract(self, inpath: Path, outdir: Path, task_result: TaskResult):
                 stdout=subprocess.PIPE,
                 stderr=subprocess.PIPE,
             )
-            if res.returncode != 0:
+            success = res.returncode == 0
```

Comment:
CommandExtractor should be parameterized as in some cases even non zero exit codes can be ok.
We should audit current CommandExtractors what is accepted as well.

Reply:
This line do not change in the final version, but yes, it is a valid future parameter.